pubsub/google/cloud/pubsub_v1/subscriber/_helper_threads.py

```diff
@@ -60,12 +60,13 @@ def __init__(self):
     def __contains__(self, needle):
         return needle in self._helper_threads
 
-    def start(self, name, queue, target):
+    def start(self, name, queue_put, target):
```

Comment:
What is the rationale behind this change?

Reply:
See where I am sending in `Consumer.send_request()` rather than passing in `Consumer._request_queue`. The problem is that if you send in a queue and then swap it off of the `Consumer`, then the helper thread won't be able to talk to the active request generator.

Reply:
Understood.

Reply:
@lukesneeringer The rationale has change slightly, since I no longer swap out the request queue. However, due to the `_put_lock` in `send_request()`, it still makes sense to avoid passing the queue itself.